modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb

```diff
@@ -48,17 +48,7 @@ def upload_supporting_documents
         end
 
         def validate_form_526
-          service = EVSS::DisabilityCompensationForm::ServiceAllClaim.new(auth_headers)
-          auto_claim = ClaimsApi::AutoEstablishedClaim.new(
-            status: ClaimsApi::AutoEstablishedClaim::PENDING,
-            auth_headers: auth_headers,
-            form_data: form_attributes
-          )
-          service.validate_form526(auto_claim.form.to_internal)
-          render json: valid_526_response
-        rescue EVSS::ErrorMiddleware::EVSSError => e
-          track_526_validation_errors(e.details)
-          render json: { errors: format_526_errors(e.details) }, status: :unprocessable_entity
+          super
```

Comment:
if you're just calling `super` do you even need this method defined?

Reply:
Yes unfortunately this is a rubocop offense in order for a method to be referenced in a before action (in this case for running validation, it needs to exist on the class that's calling it. 

```Offenses:

modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb:16:9: C: Rails/LexicallyScopedActionFilter: validate_form_526 is not explicitly defined on the class.
        before_action :validate_json_schema, only: %i[submit_form_526 validate_form_526]
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb:18:9: C: Rails/LexicallyScopedActionFilter: validate_form_526 is not explicitly defined on the class.
        before_action :validate_json_schema, only: %i[submit_form_526 validate_form_526]
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb:19:9: C: Rails/LexicallyScopedActionFilter: validate_form_526 is not explicitly defined on the class.
        before_action :validate_initial_claim, only: %i[submit_form_526 validate_form_526]```